### PR TITLE
adding industry ai prompts assets.

### PR DESCRIPTION
**Pull request by shaulimaya**
Adding industry ai prompts assests

### CODE REVIEW

File: assets/large_language_models/prompts/generate_a_discharge_letter/spec.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+$schema: https://azuremlschemas.azureedge.net/latest/asset.prompt.schema.json
+
+type: prompt
+name: generateadischargeletter
```
Comment (wayliums) [inline assets/large_language_models/prompts/generate_a_discharge_letter/spec.yaml:4]: please use - to separate, for example, generate-a-discharge-letter. Same comment apply for all other assets

File: assets/large_language_models/prompts/generate_a_discharge_letter/spec.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+version: 0.1.0
+display_name: generate_a_discharge_letter
```
Comment (wayliums) [inline assets/large_language_models/prompts/generate_a_discharge_letter/spec.yaml:6]: for display name, i don't think you need underscore anymore.

File: assets/large_language_models/prompts/one_shot_financial_news_summarization/prompt/prompt.yaml [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+- role: user
+  content: "Please summarize the following news article for financial analysts in a concise manner. Your summary should only include the most important information and avoid unnecessary words.\nnews article: Artificial Intelligence (AI) tech is an $800 billion market opportunity over the next decade as it pervades into the enterprise and consumer tech space, said Wedbush analyst Daniel Ives in a recent note.  Despite's Apple, Inc.\xE2\u20AC\u02DCs not-so-proactive approach, the analyst said Cupertino could be the \"one to benefit no matter where the industry goes.\"  Apple Preps To Press The Pedal? Given Apple is a leading player in nearly everything regarding tech and consumer technology, the company will likely step up its AI efforts over the coming months, Ives said in a recent note.  The company's thrust on AI is evident from the 2020 acquisition of Xnor.ai, a spin-off from the non-profit Allen Institute for AI, he said. The tech giant would have the opportunity to leverage Xnor.ai capabilities\
```
Comment (wayliums) [inline assets/large_language_models/prompts/one_shot_financial_news_summarization/prompt/prompt.yaml:12]: can you do the | approach for here too?

Comment (wayliums) [reply assets/large_language_models/prompts/one_shot_financial_news_summarization/prompt/prompt.yaml:12]: and can we use this style for all your multiline string?

File: assets/large_language_models/prompts/generate_a_discharge_letter/spec.yaml (hunk 2 above)
Comment (wayliums) [reply assets/large_language_models/prompts/generate_a_discharge_letter/spec.yaml:6]: and let's use something more human friendly, such as Generate a Discharge Letter

File: assets/large_language_models/prompts/one_shot_financial_news_summarization/spec.yaml [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+
+type: prompt
+name: oneshotfinancialnewssummarization
+version: 0.1.0
+display_name: one_shot_financial_news_summarization
+
+tags:
+  modality: chat
+  task: summarization
+  industry: financial_services
```
Comment (wayliums) [inline assets/large_language_models/prompts/one_shot_financial_news_summarization/spec.yaml:11]: use dash, not underscore

File: assets/large_language_models/prompts/generate_a_discharge_letter/prompt/prompt.yaml [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+    Stable. Patient experienced significant symptom relief post-PCI. No further episodes of chest pain or shortness of breath. Blood pressure under control with medications.
+- name: dob
+  description: DOB
+  defaultValue: January 1, 1960
+- name: date_of_admission
+  description: Date of Admission
+  defaultValue: June 1, 2023
+- name: date_of_discharge
+  description: Date of Discharge
+  defaultValue: June 7, 2023
```
Comment (wayliums) [inline assets/large_language_models/prompts/generate_a_discharge_letter/prompt/prompt.yaml:79]: let's rename to simply default. Sorry it was a miss from my side

File: assets/large_language_models/prompts/generate_a_discharge_letter/spec.yaml (hunk 1 above)
Comment (shaulimaya) [reply assets/large_language_models/prompts/generate_a_discharge_letter/spec.yaml]: fixed

File: assets/large_language_models/prompts/generate_a_discharge_letter/spec.yaml (hunk 2 above)
Comment (shaulimaya) [reply assets/large_language_models/prompts/generate_a_discharge_letter/spec.yaml]: fixed

File: assets/large_language_models/prompts/one_shot_financial_news_summarization/prompt/prompt.yaml (hunk 3 above)
Comment (shaulimaya) [reply assets/large_language_models/prompts/one_shot_financial_news_summarization/prompt/prompt.yaml]: fixed
The issue was related to special characters in the text that yaml package could not process right.

File: assets/large_language_models/prompts/one_shot_financial_news_summarization/spec.yaml (hunk 4 above)
Comment (shaulimaya) [reply assets/large_language_models/prompts/one_shot_financial_news_summarization/spec.yaml]: fixed

File: assets/large_language_models/prompts/generate_a_discharge_letter/prompt/prompt.yaml (hunk 5 above)
Comment (shaulimaya) [reply assets/large_language_models/prompts/generate_a_discharge_letter/prompt/prompt.yaml]: fixed

File: assets/large_language_models/prompts/parent_teacher_conference_invitation/spec.yaml [hunk 6]
```diff
@@ -0,0 +1,14 @@
+$schema: https://azuremlschemas.azureedge.net/latest/asset.prompt.schema.json
+
+type: prompt
+name: parent-teacher-conference-invitation
+version: 0.1.0
+display_name: parent teacher conference invitation
```
Comment (wayliums) [inline assets/large_language_models/prompts/parent_teacher_conference_invitation/spec.yaml:6]: per PM, let's make all display name's Title Case

Comment (shaulimaya) [reply assets/large_language_models/prompts/parent_teacher_conference_invitation/spec.yaml]: ok, fixed

**Review by alexkalita — APPROVED** [review]

**Review by wayliums — APPROVED** [review]

**Pull request merged**